src/imgui/ImGuiSpriteViewer.hh

```diff
@@ -60,6 +60,8 @@ private:
 	gl::Texture zoomGridTex{gl::Null{}};
 	gl::Texture checkerTex {gl::Null{}};
 	gl::Texture renderTex  {gl::Null{}};
+	gl::vecN<2, int> patternGridPosition;
+	gl::vecN<2, int> colorGridPosition;
```

Comment:
I didn't have time yet for a detailed review. I don't know if the following suggestion would work, or even if it is a good idea.

My thinking was: `patternGridPosition` and `colorGridPosition` are never both in use at the same time. So maybe having a single variable `gridPosition` is sufficient?